Sergey_Shashin/src/components/App.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+
+export class App extends Component {
+  state = {
+    counter: 1
+  };
+
+  handleButtonClick= (event)=>{

```

Comment:
не забывайте об аккуратном оформлении
```suggestion
  handleButtonClick= (event) => {
```